packages/react-mongo/react-mongo.ts

```diff
@@ -0,0 +1,120 @@
+import { Meteor } from 'meteor/meteor'
+import { Mongo } from 'meteor/mongo'
+import { Tracker } from 'meteor/tracker'
+import { EJSON } from 'meteor/ejson'
+import { useEffect, useMemo, useReducer, useRef, DependencyList } from 'react'
+
+const fur = (x: number): number => x + 1
+const useForceUpdate = () => useReducer(fur, 0)[1]
+
+type useSubscriptionRefs = {
+  subscription?: Meteor.SubscriptionHandle,
+  computation?: Tracker.Computation,
```

Comment:
obsolete, the computation isn't stored in the ref anymore ;-)

Reply:
(oops, actually the computation is still stored in the ref, but needlessly it seems)

Reply:
oh, I had that in the refs for some reason at one point, but you are right that doesn't need to be there any more.